.gitignore

```diff
@@ -6,3 +6,6 @@ htmlcov
 
 # msgfmt sometimes leaves these behind in the root directory
 /*.mo
+
+# IntelliJ IDE files
+.idea/
```

Comment:
This is probably a good thing to add to the `.gitignore` file, but it's unrelated to this PR. Would you mind making a new PR with just this change and removing it from this one?